plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1249,30 +1251,33 @@ read_only::get_table_by_scope_result read_only::get_table_by_scope( const read_o
 }
 
 vector<asset> read_only::get_currency_balance( const read_only::get_currency_balance_params& p )const {
+    vector<asset> results;
```

Comment:
Can we move this function to chain_api_plugin? 
It will allow to move chain_api_plugin to another utility without chain-controller.